internal/terraform/node_action_trigger_instance_plan.go

```diff
@@ -183,3 +206,97 @@ func (n *nodeActionTriggerPlanInstance) Path() addrs.ModuleInstance {
 	// to the same module. So we can simply return the module path of the action.
 	return n.actionAddress.Module
 }
+
+type conditionContext struct {
```

Comment:
```suggestion
type actionConditionContext struct {
```

It is only actions using this, so would be good to make it specific.